### PR TITLE
Crowbar: DomainName verification relaxed

### PR DESCRIPTION
**Pull request by ilausuch**
This commit allows setup more relaxed domain names folowing a more
recent RFC https://tools.ietf.org/html/rfc1123#page-13

### CODE REVIEW

File: crowbar_framework/app/models/crowbar_validator.rb [hunk 1]
```diff
@@ -92,7 +92,8 @@ def validate_hook(value, rule, path, errors)
          end
       when "DomainName"
          # http://tools.ietf.org/html/rfc1034#section-3.5
-         label = /[a-z]([a-z0-9\-]{0,61}[a-z0-9])?/i
+         # https://tools.ietf.org/html/rfc1123#page-13
+         label = /[a-z0-9]([a-z0-9\-]{0,253}[a-z0-9])?/i
```
[the remark this answers is not in the document]
Comment (ilausuch) [reply crowbar_framework/app/models/crowbar_validator.rb]: If I fix it, I have to fix in all code to have coherence. Is what we want?

**Comment by witekest** [comment]
retest this please

**Comment by witekest** [comment]
> retest this please

not sure how I can relaunch the checks

**Comment by ilausuch** [comment]
I relaunched the tests

**Pull request merged**